src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerProfile.java

```diff
@@ -89,7 +91,23 @@
 
     @ManyToOne
     @JoinColumn(name = "user_type_id", referencedColumnName = "user_type_id",
-            insertable = false, updatable = false, nullable = false)
+        insertable = false, updatable = false, nullable = false)
     private UserType userType;
 
+    @Transient
+    private boolean isNew = false;
+
+    @Override
+    public String getId() {
+        return caseWorkerId;
+    }
+
+    @Override
+    public boolean isNew() {
+        return isNew;
+    }
+
+    public void setNew(boolean isNew) {
+        this.isNew = isNew;
+    }
 }
```

Comment:
@Getter , @Setter can be used ?

Reply:
This is overridden methods Id won't smart enough to detect it and gets compilation. these are Presistable interface overrides.